doc/api/deprecations.md

```diff
@@ -2368,6 +2368,22 @@ Type: Runtime
 
 The `_stream_wrap` module is deprecated.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: modules with an invalid `main` entry and an `index.js` file
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/26823
+    description: Runtime deprecation.
+-->
+
+Type: Runtime (supports [`--pending-deprecation`][])
```

Comment:
This is not what we usually do. The type should be documentation-only as long as the default behavior does not match "Runtime". The next step would be to upgrade from `Documentation-only (supports --pending-deprecation)` to `Runtime` which should behave as if node was started with `--pending-deprecation`.

Reply:
Thanks, I had a look at the buffer constructor one and forgot that it's a special case. I updated it to `Documentation-only`.